yarn.lock

```diff
@@ -39,18 +39,18 @@
   resolved "https://registry.yarnpkg.com/@ensdomains/resolver/-/resolver-0.2.4.tgz#c10fe28bf5efbf49bff4666d909aed0265efbc89"
```

Comment:
Wow pretty large yarn.lock!

Reply:
Yeah unfortunately this is a side-effect of buidler for now.